src/renderer/pages/settings.vue

```
@@ -109,16 +109,18 @@
         :api-keys="apiKeys"
         @update:selected-l-l-m="updateSelectedLLM"
         @update:llm-configs="updateLlmConfigs"
-      />
-      <Card>
+        />
```

ESLint

[error] Delete `··`

(prettier/prettier)